### ISSUE TITLE
Should we publish the presentations we're doing in the docs?

### ISSUE THREAD

**Issue by adizere** [cometbft/cometbft-docs#56]

**Comment by alijnmerchant21** [comment]
Thanks for bringing this up Adi! I resonate with you in the idea of keeping it together in one place for easy access - especially for onboarding & easy reference. 

However, I have some hesitance when it comes to sharing it with the public. Here are some of my concerns:

- The presentation shared at conferences change very frequently, so maintaining them would be an overhead.
- We might announce things at conferences, which might be delayed more than we anticipate. It could be negative for team to not deliver in documented time. 
- Sharing it in the docs site could be - a little unclean. (Not a biggy, but just aesthetically!)

Here is my solution - we could document this in form of Blogs which could sit on CometBFT / Informal blogs on the website & medium page. Blogs are a great way of sharing updates without people expecting them to be updated. 
We could of course reference those blogs from our docs site; would be much cleaner!

**Comment by adizere** [comment]
> maintaining them would be an overhead.

Good point, I didn't think of this. I can see a simple fix: I don't think we should maintain presentations. Like an [ADR](https://github.com/cometbft/cometbft/tree/main/docs/architecture#architecture-decision-records-adr), a presentation is a snapshot or a record in time. We simply upload each presentation, tag it with the date/time, publish it, and then refer people back to it. Or am I missing something?

> We might announce things at conferences, which might be delayed more than we anticipate. It could be negative for team to not deliver in documented time.

Not sure I understand this part. Is this situation the concern?
-  _say we're at a conference and announce releasing feature X at date Y_
- _then date Y comes, but we're not ready.. now we're in a tough place because we broke our own deadline._ 

How does this situation relate to uploading the slides?

> Sharing it in the docs site could be - a little unclean. (Not a biggy, but just aesthetically!)

I imagine it would be a simple download button. Agree it's not idea because there's no "integration" in the website. @aaronmw any thoughts here? We could publish the PDFs at [slideshare](https://www.slideshare.net/aniruddha.chakrabarti/golang-82668216) or [slides.com](https://slides.com/peterstace/golang-presentation).

**Comment by alijnmerchant21** [comment]
> We simply upload each presentation, tag it with the date/time, publish it, and then refer people back to it.

On the documentation site, there is a section called Past Presentations (TBD) and in that, we have all PDFs linked there to be downloaded or Slideshare link - Is that what you mean by uploading each presentation?

> then date Y comes, but we're not ready.. now we're in a tough place because we broke our own deadline.

My concern was if we had the deadline documented at cometbft docs and did not adhere to it, it would be negative. However, if it's a pdf that is to be downloaded, then it is not a concern.

**Comment by adizere** [comment]
Thanks for the feedback and concerns Ali!

> On the documentation site, there is a section called Past Presentations (TBD) and in that, we have all PDFs linked there to be downloaded or Slideshare link - Is that what you mean by uploading each presentation?

Yes.

Curious if @aaronmw has any thoughts here. Otherwise I think we should go ahead with this in Q2.

To be clear, I don't want to create additional work for us. Instead, this issue seeks to achieve the publishing of work we're doing anyway (presentations at conference, hackatons, demos) with a minimal overhead.
